### PR TITLE
[CodeStyle][Typos][H-[1-9]] Fix typos(`Halfs`,`Hanlder`,`hander`,`Hander`,`hadler`,`happend`,`hardward`,`hight`,`helpe`,`hepler`,`hge`,`homogenous`,`horizonal`)

### PR DESCRIPTION
**Pull request by enkilee**
### PR Category

User Experience

### PR Types

Devs

### Description

fixed:
`Halfs`,
`Hanlder`,
`hander`,
`Hander`,
`hadler`,
`happend`,
`hardward`,
`hight`,
`helpe`,
`hepler`,
`homogenous`,
`horizonal`

skiped:
`hge` : cause cuda used it:
`cuda_fp16.h`:
```bash
__host__ __device__ bool __hge(const __half a, const __half b)
```

### CODE REVIEW

File: _typos.toml [hunk 1]
```diff
@@ -253,6 +242,7 @@ localy = 'localy'
 Localy = 'Localy'
 logarithmical = 'logarithmical'
 Loggin = 'Loggin'
+Maxinum = 'Maxinum'
```
Comment (SigureMo) [inline _typos.toml]: 删掉，#70474 已经修改

**Review by SigureMo — COMMENTED** [review]
其余 LGTM

File: _typos.toml (hunk 1 above)
Comment (enkilee) [reply _typos.toml]: 收到

**Review by SigureMo — APPROVED** [review]
LGTMeow

**Pull request merged**